content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: '.getOwnPropertyNames()'
+Description: 'Returns an array of strings that corresponds to the properties found directly in the given object.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Objects'
+  - 'Methods'
+  - 'Values'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`Object.getOwnPropertyNames()`** method returns all properties that are present in a given object `obj` except for those symbol-based non-enumerable properties.

```

Comment:
```suggestion
The **`obj.getOwnPropertyNames()`** method returns all properties that are present in a given object `obj` except for those symbol-based non-enumerable properties.
```
We referring to this object as `obj` in the sentence so let's change it to that,